ui/src/shared/graphs/helpers.js

```diff
@@ -77,12 +85,14 @@ export const multiColumnBarPlotter = e => {
         yBottom - p.canvasy
       )
 
-      ctx.strokeRect(
-        xLeft,
-        p.canvasy,
-        barWidth / sets.length,
-        yBottom - p.canvasy
-      )
+      if (selPointX === centerX) {
```

Comment:
what is this doing? something related to determining which point should be displayed as selected when highlighting?